arbitrum-docs/node-running/troubleshooting-running-nodes.md

```diff
@@ -1,9 +1,179 @@
+    <div className='guidance-container'>
+        <label htmlFor="st-1">4. TODO - this changes with the OS</label>
+        <Tabs className="tabgroup-with-label" groupId="os" defaultValue="others" values={[
+            {label: 'Operating system:', value: 'label'},
+            {label: 'Linux, MacOS, Arm64', value: 'others'},
+            {label: 'Windows', value: 'win'}
+            ]}>
+            <TabItem className="unclickable-element" value="label"></TabItem>
+            <TabItem value="others"><p>Demo -  others</p></TabItem>
+            <TabItem value="win"><p>Demo -  windows</p></TabItem>
+        </Tabs>
+    </div>
+</div>
+<div className='task'>
+    <div className='input-container'><input id="st-2" type='checkbox'/><span className='done'></span></div>
+    <div className='guidance-container'>
+        <label htmlFor="st-2">5. TODO - this changes with the network</label>
+        <Tabs className="tabgroup-with-label network-tabgroup" groupId="network" defaultvalue="arb-one" values={[   {label: 'Network:', value: 'label'},
+                    {label: 'Localhost', value: 'localhost'},
+                    {label: 'Arbitrum Goerli', value: 'arb-goerli'},
+                    {label: 'Arbitrum One', value: 'arb-one'},
+                    {label: 'Arbitrum Nova', value: 'arb-nova'}
+                ]}>
+            <TabItem className="unclickable-element" value="label"></TabItem>
+            <TabItem value="localhost"><p>Demo -  localhost</p></TabItem>
+            <TabItem value="arb-goerli"><p>Demo -  goerli</p></TabItem>
+            <TabItem value="arb-one"><p>Demo -  one</p></TabItem>
+            <TabItem value="arb-nova"><p>Demo -  nova</p></TabItem>
+        </Tabs>
+    </div>
+</div>
+-->
+
+
+### Step 2: Look for your scenario
+
+Common troubleshooting scenarios and solutions are detailed below.
+
+<table className='small-table'>
+  <tbody>
+      <tr>
+          <th style={{minWidth: 180 + 'px'}}>Scenario</th> 
+          <th>Solution</th>
+      </tr>
+      <tr>
+        <td>You see <code>Unindex transactions</code>.</td>
+        <td>This is expected behavior. You'll see this when your node removes old <code>txlookup</code> indices. This is emitted from the base Geth node, so you'd see the same output from a mainnet Geth node.</td>
+      </tr>
+      <tr>
+        <td>You see <code>Head state missing, repairing</code>.</td>
+        <td>This is usually because your node shut down ungracefully. In most cases it will recover in few minutes, but if it not, you may have to re-sync your node. Remember to shut down your node gracefully with the following command: <code>docker stop —time=300 $(docker ps -aq)</code>.</td>
+      </tr>
+      <tr>
+        <td>You see <code>failed to read inbox messages</code></td>
+        <td>This is usually because your L1 RPC is unreachable. Check your L1 RPC status or use another L1 RPC.</td>
```

Comment:
Sometimes this also because the l1 node hasn't finished syncing, so it will read an old l1 node state which doesn't have our inbox contracts deployed.